packaging/tags.py

```diff
@@ -407,7 +407,12 @@ def _mac_binary_formats(version, cpu_arch):
             return []
         formats.extend(["fat32", "fat"])
 
-    formats.append("universal")
+    elif cpu_arch == "arm64":
+        formats.append("universal2")
+
+    if cpu_arch != "arm64":
```

Comment:
```suggestion
    if cpu_arch in {"x86_64", "i386", "ppc64", "ppc"}:
```

Prevent appending "universal" for unknown arch (who knows what will come next)